qp_shogun/utils.py

```diff
@@ -157,7 +157,15 @@ def _per_sample_ainfo(
         for suff in suffixes:
             fname = smd(suff % rp)
             if exists(fname):
-                files.append(fname)
+                if fname.endswith('R1.fastq.gz'):
+                    ftype = 'raw_forward_seqs'
+                elif fname.endswith('R2.fastq.gz'):
+                    ftype = 'raw_reverse_seqs'
+                else:
+                    # this should never happen and it's not really possible
+                    # to reproduce so no tests!
```

Comment:
If one were to test this case, could you create some empty files in the test code that did exist but  didn't have one of those endings? 

Maybe change from 'not really possible to reproduce' to "TODO?" I could also write one up and submit a PR to your branch